<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administrador_tarjeta_cancelacion_cita</class>
 <widget class="QWidget" name="administrador_tarjeta_cancelacion_cita">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_prioridad">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>95</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>Prioridad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_horario_admin_cancelar_cita">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;
</string>
   </property>
   <property name="text">
    <string>17:30:00 - 18:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_prioridad_admin_cancelar_cita">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>95</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/imagenes/Imagenes/Ellipse7.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_foto_admin_cancelar_cita">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>24</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/imagenes/Imagenes/imgPrueba.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_motivo_admin_cancelar_cita">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>45</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>Dolor de cabeza</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>359</width>
     <height>149</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F4F4F4;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_admin_visualizar_cancelacion_cita">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
   </property>
   <property name="text">
    <string>Visualizar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nombre_admin_tarjeta_horario">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>Luis Rolando Banderas Barrera</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>idCita:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_idCita_admin_cancelar_cita">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>09</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>btn_admin_visualizar_cancelacion_cita</zorder>
  <zorder>lbl_nombre_admin_tarjeta_horario</zorder>
  <zorder>lbl_foto_admin_cancelar_cita</zorder>
  <zorder>lbl_horario_admin_cancelar_cita</zorder>
  <zorder>lbl_motivo_admin_cancelar_cita</zorder>
  <zorder>lbl_prioridad_admin_cancelar_cita</zorder>
  <zorder>lbl_prioridad</zorder>
  <zorder>label_2</zorder>
  <zorder>lbl_idCita_admin_cancelar_cita</zorder>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
